Cargo.toml

```
@@ -12,7 +12,7 @@ cortex-m = "0.7"
 cortex-m-rt = "0.6"
 panic-reset = "0.1"
 nb = "1"
-panel-protocol = { git = "https://github.com/tonarino/panel-protocol.git", rev = "0.4" }
+panel-protocol = { git = "https://github.com/tonarino/panel-protocol.git", branch = "const-generics" }
```

Hmm I thought we've merged and tagged `const-generics`?

---

Had that comment almost sent. Yes we did, we can use 
```suggestion
panel-protocol = { git = "https://github.com/tonarino/panel-protocol.git", rev = "0.6" }
```

---

Oh yeah I think we have. I just realized I never opened a PR for the branch I pushed so I wanted to open it before I forgot.

---

I pushed a commit that changes this back to `rev` reference.

@bschwind or @strohel feel free to merge I guess.